keyboards/xd75/keymaps/cryptokid91/keymap.c

```diff
@@ -0,0 +1,179 @@
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+ * |        |        |        |        |        |        |        |        |        |        |        |        |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * |        |        |        |        |        |        |        | RESET  |        |        |        |        |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * |        |        |        |        |        |        |        |        |        |        |        |        |        |        |        |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * |        |        |        |        |        |        |        |        |        |        |        |        |        |        |        |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+ 
+ [_RESET] = LAYOUT_ortho_5x15( /* RESET */
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+   _______, _______, _______, _______, _______, _______, _______, RESET,   _______, _______, _______, _______, _______, _______, _______,
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+ )
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+  case MS_P00:
+    if (record->event.pressed) {
+      SEND_STRING("00");
+    }
+    return false;
+  case NEWTAB:
+    if(record->event.pressed) {
+      SEND_STRING(SS_LCTRL("c") SS_LCTRL("t"));
+      wait_ms(100);
+      SEND_STRING(SS_LCTRL("v"));
+      wait_ms(100);
+      SEND_STRING(SS_TAP(X_ENTER));
+    }
+    return false;
+  case ROLL:
+    if(record->event.pressed) {
+      SEND_STRING("!roll 1d100\n");
+    }
+    return false;
+  case IOTECH:
+    if(record->event.pressed) {
+      SEND_STRING(SS_LCTRL("t"));
+      wait_ms(100);
+      SEND_STRING("www.io/tech.fi");
+      wait_ms(100);
+      SEND_STRING(SS_TAP(X_ENTER));
+    }
+    return false;
+  case TERM:
+    if(record->event.pressed) {
+      SEND_STRING(SS_LGUI("r"));
+      wait_ms(300);
```

Comment:
I think we just added a "delay" function to SEND_STRING.  

https://docs.qmk.fm/#/feature_macros?id=tap-down-and-up

```c
SS_DELAY(msecs)
```

So you could make one long line here, if you want.  Also, `\n` should trigger a line return.